osu.Game/Screens/Play/Player.cs

```diff
@@ -345,6 +346,18 @@ public override void OnEntering(IScreen last)
                 .Delay(250)
                 .FadeIn(250);
 
+            ShowStoryboard.ValueChanged += s =>
+            {
+                if (s.NewValue && storyboard == null)
+                    initializeStoryboard(true);
+            };
+
+            Background.EnableUserDim.Value = true;
+
+            storyboardReplacesBackground.BindTo(Background?.StoryboardReplacesBackground);
+            storyboardReplacesBackground.BindTo(storyboardContainer.StoryboardReplacesBackground);
+            storyboardReplacesBackground.Value = Beatmap.Value.Storyboard.ReplacesBackground && Beatmap.Value.Storyboard.HasDrawable;
```

Comment:
as above

Reply:
Not quite sure what this is referring to. It's setting the value as a one-off thing, not binding to a bindable.

Reply:
already fine, was attached to the second BindTo line before